### PR TITLE
Enhance contributing guide

### PR DESCRIPTION
**Pull request by heitorPB**
Improve text in the contributing guide around git usage and the release process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,81 +48,39 @@
...
     ```sh
-    git clone https://github.com/your_username_/Charm-Name.git
+    $ git clone https://github.com/omnivector-solutions/osd-documentation.git
     ```
 1. Install the dependencies:
     ```sh
-    virtualenv venv
-    source venv/bin/activate
-    pip install -r requirements.txt
+    $ python3 -m venv venv
```
Comment (omnivector-kelly) [inline README.md:72]:
```suggestion
    $ cd ./osd-documentation
    $ python3 -m venv venv
```

File: contributing.rst [hunk 2]
```diff
@@ -114,14 +124,18 @@ Git usage
...
     needed
+  - See `How to Write a Git Commit Message <https://cbea.ms/git-commit/>`_ for
+    a detailed guide on this subject
 
 - linear history
 
-  - always rebase with master
-  - fast forward merge or squash merge
+  - always rebase your branch with master
+  - use fast forward merge or squash merge, when merging the Pull Requests
```
Comment (omnivector-kelly) [inline contributing.rst:133]:
```suggestion
  - When merging the Pull Requests, use fast forward merge or squash merge.
```

File: contributing.rst [hunk 3]
```diff
@@ -311,6 +325,10 @@ The Slurm-charms and ``slurm-ops-manager`` follow a semantic versioning scheme.
 Release process
 ---------------
 
+The release process is a different process than writing code to add new
+features or to fix issues. Crafting a new release means building a new version
+of the product and making it available to clients and end users.
+
```
Comment (omnivector-kelly) [inline contributing.rst:331]:
```suggestion
The release process is separated from the development process.
The development process includes writing code to add features and or
fixing issues. The release process involves building a new version of the
product and making it available to clients and end users.
```
Suggesting edit for reading flow.

Comment (heitorPB) [reply contributing.rst:331]: That is much better!

**Review by derekcat — APPROVED** [review]
Looks good to me

**Review by omnivector-kelly — APPROVED** [review]
Awesome Work.

**Pull request merged**